torch/_inductor/scheduler.py

```diff
@@ -927,6 +927,7 @@ def should_prune(dep: Dep) -> bool:
     "extern_kernels.mm": torch.ops.aten.mm,
     "extern_kernels.bmm": torch.ops.aten.bmm,
     "extern_kernels.addmm": torch.ops.aten.addmm,
+    "extern_kernels._scaled_mm": torch.ops.aten._scaled_mm,
```

Comment:
cc @xmfan theres a todo here from you. 

> # TODO(xmfan): reuse: an existing mapping for this if it exists, or formalize this into ir.py:ExternKernel

we should able to reuse existing FlopCounter registrations without this list here